parts/kubernetesmastercustomdata.yml

```diff
@@ -335,7 +338,57 @@ write_files:
     fi
     mount $MOUNTPOINT
 
+{{if .OrchestratorProfile.IsCustomEtcdVersion}}
+- path: "/etc/systemd/system/etcd.service"
+  permissions: "0644"
+  owner: "root"
+  content: |
+    [Unit]
+    Description=etcd - highly-available key value store
+    Documentation=https://github.com/coreos/etcd
+    Documentation=man:etcd
+    After=network.target
+    Wants=network-online.target
+    [Service]
+    Environment=DAEMON_ARGS=
+    Environment=ETCD_NAME=%H
+    Environment=ETCD_DATA_DIR=/var/lib/etcd/default
```

Comment:
Are these still valid and recommend for k8s in etcd3

Reply:
All of this gets override-en with env file where data file is actually saved on a mounted data disk

Reply:
Okay let me spin this up and take a look.